CONTRIBUTING.md

```diff
@@ -2,8 +2,18 @@
 
 Each proposal to Mainnet (agoric-3) should be contributed to this repo before being proposed to the chain.
 
-1. Prepare the proposal and verify it running locally
+1. Prepare the proposal and verify it by creating a proposal in
+   agoric-sdk/a3p-integration. See the README there for more details
+
 2. Submit a PR to this repo in which the proposal has a _pending_ name (e.g. `a:my-contract`)
+
 3. Get review and merge to main
+
 4. When the proposal has been further qualified with higher fidelity to Mainnet, submit to Mainnet
-5. When the proposal passes, submit a PR here converting the pending proposal to a passed one (by changing the letter to the submission number)
+
+5. When the proposal passes, submit a PR here converting the pending proposal to a passed one (by
```

Comment:
this is showing as an edit but I don't see the change.

Reply:
Added a line break.